crates/shared/src/http_solver.rs

```diff
@@ -173,6 +177,37 @@ impl HttpSolverApi for DefaultHttpSolverApi {
         serde_json::from_str(text)
             .with_context(|| format!("failed to decode response json, {}", context()))
     }
+
+    fn notify_auction_result(&self, auction_id: AuctionId, result: model::AuctionResult) {
+        let mut url = match self.base.join("notify") {
+            Ok(url) => url,
+            Err(err) => {
+                tracing::warn!(?err, "failed to create notify url");
+                return;
+            }
+        };
+
+        let client = self.client.clone();
+        let config_api_key = self.config.api_key.clone();
+        let future = async move {
+            url.query_pairs_mut()
+                .append_pair("auction_id", auction_id.to_string().as_str());
```

Comment:
I think Poolpi was asking to also set the auction_id as a header. Not sure if this is necessary though.

Reply:
Was not sure about this, but followed the example of `solve` for consistency (personally I think it doesn't make any important difference). Asked @poolpitako for this one.

Reply:
Confirmed it is ok with query param.